portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,28 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.content {
```

Comment:
From https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order: "Sort declarations consistently within a project. In the absence of tooling to automate and enforce a consistent sort order, consider putting declarations in alphabetical order in order to achieve consistent code in a way that is easy to learn, remember, and manually maintain."